### ISSUE TITLE
[Bug] No regions in dropdown

### ISSUE THREAD

**Issue by PashaKagan** [redphx/better-xcloud#411]
### Checklist

- [X] I will only use English in my report.
- [X] The bug doesn't happen when I disable Better xCloud script.
- [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### Device

Desktop

### Device

PC

### Operating System

Windows 11

### Browser Version

Edge 126.0.2592.24 

### Better xCloud Version

4.5.0

### Reproduction Steps

Whenever I try to access the region dropdown, it appears empty. 
Additionally, question marks appear where the region is supposed to be - maybe it has something to do with the issue.

### Screenshot/video

![image](https://github.com/redphx/better-xcloud/assets/8081472/7b97cd7f-7ae9-4c68-a8da-6b34ef8f42bb)
![image](https://github.com/redphx/better-xcloud/assets/8081472/fbc99904-ab53-44a7-add0-31ad57b7108a)

**Comment by redphx** [comment]
Check this https://better-xcloud.github.io/kiwi-browser/#troubleshooting

**Comment by PashaKagan** [comment]
Thank you, but I'm on Edge and it is in fact installed correctly. Verified with several PCs.
Is there something else that could affect this?

**Comment by redphx** [comment]
share screenshots of chrome://extensions & Tampermonkey's dashboard

**Comment by PashaKagan** [comment]
![image](https://github.com/redphx/better-xcloud/assets/8081472/5d4a2947-1921-4d75-8c4f-d23bd20abc9a)
![image](https://github.com/redphx/better-xcloud/assets/8081472/67b8f90f-66f3-44a0-a363-d1d071df513b)

**Comment by redphx** [comment]
Try Tampermonkey. This script isn't tested with Violentmonkey.

**Comment by PashaKagan** [comment]
It's not the cause, I figured it out. You were in a hurry to close the ticket :)

It has to do with my region, where I live. I use VPN for work - and I noticed that when I'm logged in (regionally, in the USA), it does work, **with** Violentmonkey and the same config.

![image](https://github.com/redphx/better-xcloud/assets/8081472/4dedc010-d5b2-476e-a1df-ee6f9f838eb4)
![image](https://github.com/redphx/better-xcloud/assets/8081472/f0b401ad-fe37-4d72-a724-086327cda846)

To conclude - doesn't work in my own region, does work when I'm somewhere else.

Isn't the script supposed to help with that?
